### DISCUSSION TITLE
Table V7 in-cell stacked (aggregated) gauges

### DISCUSSION THREAD

**Discussion by jayemce** [grafana/grafana#38394] [Dashboards, Panels & Visualizations]
Referencing https://github.com/grafana/grafana/issues/25969

**What would you like to be added**:
We adopted the table in-cell gauge feature in our compliance reporting results to highlight the difference in compliance in % values. The capability of stacking values from two different cells would allow for a graphing like in the example I'm making below.

This is one of our tables today with two in-cell gauges:
![tableV7stackingrequest](https://user-images.githubusercontent.com/11334355/86387198-182c6b00-bc93-11ea-8bc3-f1586af4ac26.png)

This is how it could look like with in-cell stacked gauges:
![tableV7stackingEnhancementSimulation](https://user-images.githubusercontent.com/11334355/86385831-32654980-bc91-11ea-8183-6e156d36c41d.png)

**Why is this needed**:
Eye catching when comparing related metrics.
Space saving.

**Comment by torkelo** [comment]
Do not think it will be possible without requiring to complex options and implemention. Would have to be custom panel of sorts that just does that

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
